docs/content/Metrics.md

```diff
@@ -3,11 +3,182 @@ layout: doc_page
+
+### Historical/Real-time
+
+|Metric|Description|Normal Value|
+|--------|-----------|-------|
+|`query/time`|Milliseconds taken to complete a query.|< 1s|
+|`query/segment/time`|Milliseconds taken to query individual segment. Includes time to page in the segment from disk.|< several hundred milliseconds|
+|`query/wait/time`|Milliseconds spent waiting for a segment to be scanned.|< several hundred milliseconds|
+|`segment/scan/pending`|Number of segments in queue waiting to be scanned.|Close to 0|
+|`query/segmentAndCache/time`|Milliseconds taken to query individual segment or hit the cache (if it is enabled on the historical node).|< several hundred milliseconds|
+
+### Cache
+
+|Metric|Description|Normal Value|
+|--------|-----------|-------|
+|`query/cache/delta/*`|Cache metrics since the last emission.|N/A|
+|`query/cache/total/*`|Total cache metrics.|N/A|
+
+
+|Metric|Description|Normal Value|
+|--------|-----------|-------|
+|`*/numEntries`|Number of cache entries.|Varies.|
+|`*/sizeBytes`|Size in bytes of cache entries.|Varies.|
+|`*/hits`|Number of cache hits.|Varies.|
+|`*/misses`|Number of cache misses.|Varies.|
+|`*/evictions`|Number of cache evictions.|Varies.|
+|`*/hitRate`|Cache hit rate.|~40%|
+|`*/averageByte`|Average cache entry byte size.|Varies.|
+|`*/timeouts`|Number of cache timeouts.|0|
+|`*/errors`|Number of cache errors.|0|
+
+## Ingestion Metrics
+
+|Metric|Description|Normal Value|
+|--------|-----------|-------|
+|`ingest/events/thrownAway`|Number of events rejected because they are outside the windowPeriod.|0|
+|`ingest/events/unparseable`|Number of events rejected because the events are unparseable.|0|
+|`ingest/events/processed`|Number of events successfully processed.|Equal to your # of events.|
+|`ingest/rows/output`|Number of Druid rows persisted.|Your # of events with rollup.|
+|`ingest/persists/count`|Number of times persist occurred.|Depends on configuration.|
+|`ingest/persists/time`|Milliseconds spent doing intermediate persist.|Depends on configuration.|Generally a few minutes at most.|
+|`ingest/persists/backPressure`|Number of persists pending.|0|
+|`ingest/persists/failed`|Number of persists that failed.|0|
+|`ingest/handoff/failed`|Number of handoffs that failed.|0|
+
+### Indexing Service
+
+|Metric|Description|Normal Value|
+|--------|-----------|-------|
+|`task/run/time`|Milliseconds taken to run task.|Varies.|
+|`segment/added/bytes`|Size in bytes of new segments created.|Varies.|
+|`segment/moved/bytes`|Size in bytes of segments moved/archived via the Move Task.|Varies.|
+|`segment/nuked/bytes`|Size in bytes of segments deleted via the Kill Task.|Varies.|
+
+## Coordination
+
+These metrics are for the Druid coordinator and are reset each time the coordinator runs the coordination logic.
+
+The following metrics are emitted on a per tier basis:
+
+|Metric|Description|Normal Value|
+|--------|-----------|-------|
+|`segment/*/added/count`|Number of segments added to the cluster.|Varies.|
```

Comment:
can we put "tier" as a dimension instead of encoding it in the metric name as is done for some other metrics?